tensornetwork/tensornetwork_test.py

```diff
@@ -1508,6 +1509,98 @@ def test_split_node_rq(backend):
   np.testing.assert_allclose(a.tensor, net.contract(left[3]).tensor)
 
 
+def test_split_node_qr_unitarity_complex_numpy():
```

Comment:
Please you the `backend` argument to reduce code copying.

Reply:
can do as soon as we have backend.conj() (see #213)